### DISCUSSION TITLE
How can I set IdleSettings.WaitTimeout to TimeSpan.Zero?

### DISCUSSION THREAD

**Discussion by s-lazukov** [dahall/TaskScheduler#903] [Q&A] [answered]
In the Task Scheduler I can set the "Wait for idle for:" parameter to "Do not wait". How can I set this value in code?
If I set IdleSettings.WaitTimeout to TimeSpan.Zero, then the value is not written to the task parameters and the TaskScheduler uses the default value (1 hour)

Thanks,
Sergey

**Comment by dahall** [comment, marked as answer]
Thanks Sergey. After almost 10 years with no bugs, you found one. I'll make the fix so that `TimeSpan.Zero` will represent "Do not wait". Moving this to an issue so I can track it.

**Reply by s-lazukov** [reply]
Thank you, looking forward to the fix.

**Comment by dahall** [comment]
I published it about 10 hours ago as version 2.9.2.

**Reply by s-lazukov** [reply]
Thank you
